docs/DynamicPartitionedORCFileSet-batchsink.md

```diff
@@ -0,0 +1,56 @@
+# Dynamic PartitionedFileSet ORC Batch Sink
+
+
+Description
+-----------
+Sink for a ``PartitionedFileSet`` that writes data in ORC format
+and leverages one or more record field values for creating partitions.
+All data for the run will be written to a partition based on the
+specified fields and their value.
+
+
+Use Case
+--------
+This sink is used whenever you want to write to a ``PartitionedFileSet`` in ORC format
+using a value from the record as a partition. For example, you might want to load historical
+data from a database and partition the dataset on the original creation date of the data.
+
+
+Properties
+----------
+**name:** Name of the ``PartitionedFileSet`` to which records are written.
+If it doesn't exist, it will be created.
+
+**schema:** The Avro schema of the record being written to the sink as a JSON Object.
+
+**basePath:** Base path for the ``PartitionedFileSet``. Defaults to the name of the dataset.
+
+**compressionCodec:** Optional parameter to determine the compression codec to use on the resulting data. 
+Valid values are None, Snappy, GZip, and LZO.
+
+Example
+-------
+This example will write to a ``PartitionedFileSet`` named ``'users'`` using the original
+``'create_date'`` of the record as the partition:
+
+    {
+        "name": "DynamicPartitionedORCFileSet",
```

Comment:
So do we remove the whole Example. Or should we just replace the json with something else?